cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -439,12 +471,14 @@ externalSetupMethod path verbosity options _ args = do
       searchpath <- programSearchPathAsPATHVar
                     (map ProgramSearchPathDir (useExtraPathEnv options) ++
                       getProgramSearchPath (useProgramDb options))
-      env        <- getEffectiveEnvironment [("PATH", Just searchpath)
+      env'        <- getEffectiveEnvironment [("PATH", Just searchpath)
                                             ,("HASKELL_DIST_DIR", Just (useDistPref options))]
 
-      process <- runProcess path' args
-                  (useWorkingDir options) env Nothing
+      -- TODO use createProcess
```

Comment:
The only reason I defined a runProcess' was to deduplicate code actually.  But I guess I could inline it

That TODO was meant to be removed.